MetadataMigration/src/main/java/org/opensearch/migrations/MetadataMigration.java

```diff
@@ -0,0 +1,49 @@
+package org.opensearch.migrations;
+
+import org.opensearch.migrations.commands.Configure;
+import org.opensearch.migrations.commands.Evaluate;
+import org.opensearch.migrations.commands.Migrate;
+import org.opensearch.migrations.metadata.tracing.RootMetadataMigrationContext;
+import org.opensearch.migrations.tracing.ActiveContextTracker;
+import org.opensearch.migrations.tracing.ActiveContextTrackerByActivityType;
+import org.opensearch.migrations.tracing.CompositeContextTracker;
+import org.opensearch.migrations.tracing.RootOtelContext;
+
+import com.beust.jcommander.JCommander;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class MetadataMigration {
+
+    public static void main(String[] args) throws Exception {
+        MetadataArgs arguments = new MetadataArgs();
+        JCommander.newBuilder().addObject(arguments).build().parse(args);
+
+        var context = new RootMetadataMigrationContext(
+            RootOtelContext.initializeOpenTelemetryWithCollectorOrAsNoop(arguments.otelCollectorEndpoint, "rfs"),
```

Comment:
I think this was possibly addressed in a recent PR, but we should set the name here to "metadata" or something similar to ensure these traces are distinguishable from the document migration ones